cockroachdb/manifests-cert-manager/scripts.yaml

```diff
@@ -16,7 +15,27 @@ data:
         echo "DB_NAME envvar not set"
         exit 1
     fi
+    # Wait for cluster to be ready
```

Comment:
could we move this to a lifecycle hook on the container, and let k8s handle the retrying/back-off etc. behaviour?

Why doesn't the main cockcorach container in the STS suffer from the same issues, can we just re-use whatever that uses to figure out 'readiness'?

I'm also not a fan of this just being copied across the two scripts, it feels like it would be very easy to update one and forget to update the other

Reply:
Lifecycle hooks execute synchronously with container entrypoint. The same about readiness probes.

Trying to make it nicer, I put it into separate script that runs before main scripts. Not sure if it's much better, but I can't find a better way to do it.